### ISSUE TITLE
[Component] [tree-select] el-tree-select 无法获取焦点

### ISSUE THREAD

**Issue by 3000-codes** [element-plus/element-plus#13825]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.3.4`
- Element Plus Version: `2.3.8`
- Browser / OS: ` Chrome114.0.0.0 win11`
- Build Tool: `Vite`

### Related Component

- `el-tree-select`

### Reproduction Link

[Link]()

### Steps to reproduce

```vue
<script setup>
import { ref } from "vue";
import { ElTreeSelect} from "element-plus";
const data = [
    {
        value: "1",
        label: "Level one 1",
        children: [
            {
                value: "1-1",
                label: "Level two 1-1",
                children: [
                    {
                        value: "1-1-1",
                        label: "Level three 1-1-1"
                    }
                ]
            }
        ]
    },
    {
        value: "2",
        label: "Level one 2",
        children: [
            {
                value: "2-1",
                label: "Level two 2-1",
                children: [
                    {
                        value: "2-1-1",
                        label: "Level three 2-1-1"
                    }
                ]
            },
            {
                value: "2-2",
                label: "Level two 2-2",
                children: [
                    {
                        value: "2-2-1",
                        label: "Level three 2-2-1"
                    }
                ]
            }
        ]
    }
];

const form = ref({
    ids: []
});

const handleFocus = () => {
    console.log("focus");
};

const handleBlur = () => {
    console.log("blur");
};
</script>
<template>
  <el-form ref="formRef" :model="form">
            <el-form-item label="test">
                <el-tree-select
                    v-model="form.ids"
                    :data="data"
                    show-checkbox
                    check-strictly
                    check-on-click-node
                    filterable
                    multiple
                    @focus="handleFocus"
                    @blur="handleBlur"
                    ref="selectTreeRef"
                />
            </el-form-item>
        </el-form>
</template>
```

### What is Expected?

点击下拉箭头,el-tree正确弹出

### What is actually happening?

点击输入框的下拉箭头时,输入框触发两次获取焦点,很快又触发两次失去焦点,同时el-tree-select中的el-tree未弹出

### Additional comments

仅在同时使用filterable和multiple这两个属性时触发

**Comment by btea** [comment]
Should be fixed in #13699

**Comment by 3000-codes** [comment]
it fixed in https://github.com/element-plus/element-plus/pull/13699
